app/models/course_user.rb

```diff
@@ -89,6 +89,8 @@ class CourseUser < ActiveRecord::Base
       order('achievement_count DESC, last_obtained_achievement ASC')
   end)
 
+  scope :order_alphabetically, ->(direction = :asc) { order(name: direction) }
```

Comment:
I imitated these:

```
app/models/course/assessment/submission.rb
95:  scope :ordered_by_date, ->(direction = :desc) { order(created_at: direction) }

app/models/instance.rb
58:  scope :order_by_id, ->(direction = :asc) { order(id: direction) }

app/models/system/announcement.rb
8:  scope :ordered_by_date, ->(direction = :desc) { order(start_at: direction) }
```

Maybe we should standardize whether to use past tense "ordered" or the imperative "order". Which should it be?